membersrvc/ca/ca.go

```diff
@@ -340,6 +345,9 @@ func (ca *CA) createCertificate(id string, pub interface{}, usage x509.KeyUsage,
 }
 
 func (ca *CA) createCertificateFromSpec(spec *CertificateSpec, timestamp int64, kdfKey []byte) ([]byte, error) {
+	mutex.Lock()
```

Comment:
@jeronimoirazabal Just a clarification, can't we remove this lock by having a db transaction wrapping the sql insert? SQLite3 should support synchronization automatically, no?